app/assets/javascripts/components/client/RegisterClient.jsx

```diff
@@ -0,0 +1,183 @@
+class RegisterClient extends React.Component {
```

Comment:
add some comments at the top describing the props passed into this component just for documentation purposes (see: https://github.com/calblueprint/sjc/blob/master/app/assets/javascripts/components/client/ClientComments.jsx)